tests/foreman/api/test_errata.py

```diff
@@ -20,49 +20,28 @@
 from time import sleep
 
 import pytest
+from broker import VMBroker
 from nailgun import entities
 
+from robottelo import constants
 from robottelo.api.utils import enable_rhrepo_and_fetchid
 from robottelo.api.utils import promote
 from robottelo.cli.factory import setup_org_for_a_custom_repo
 from robottelo.cli.factory import setup_org_for_a_rh_repo
 from robottelo.config import settings
-from robottelo.constants import DEFAULT_ARCHITECTURE
-from robottelo.constants import DEFAULT_RELEASE_VERSION
-from robottelo.constants import DISTRO_RHEL6
-from robottelo.constants import DISTRO_RHEL7
-from robottelo.constants import DISTRO_RHEL8
-from robottelo.constants import FAKE_1_CUSTOM_PACKAGE
-from robottelo.constants import FAKE_1_CUSTOM_PACKAGE_NAME
-from robottelo.constants import FAKE_2_CUSTOM_PACKAGE
-from robottelo.constants import FAKE_2_ERRATA_ID
-from robottelo.constants import FAKE_3_ERRATA_ID
-from robottelo.constants import FAKE_3_YUM_ERRATUM_COUNT
-from robottelo.constants import FAKE_9_YUM_ERRATUM
-from robottelo.constants import FAKE_9_YUM_ERRATUM_COUNT
-from robottelo.constants import FAKE_9_YUM_OUTDATED_PACKAGES
-from robottelo.constants import PRDS
-from robottelo.constants import REAL_0_ERRATA_ID
-from robottelo.constants import REAL_0_RH_PACKAGE
-from robottelo.constants import REAL_1_ERRATA_ID
-from robottelo.constants import REAL_2_ERRATA_ID
-from robottelo.constants import REPOS
-from robottelo.constants import REPOSET
-from robottelo.constants.repos import CUSTOM_SWID_TAG_REPO
-from robottelo.constants.repos import FAKE_3_YUM_REPO
-from robottelo.constants.repos import FAKE_9_YUM_REPO
+from robottelo.constants import repos
```

Comment:
What's the reason for this change?

Reply:
IMO it looks better to have one import instead of several from the same location. it is also more explicit where those constants come from when used in code (more clear when something is a local/vs imported constant).